lib/zuora/objects/refund_invoice_payment.rb

```diff
@@ -1,18 +1,18 @@
-#module Zuora::Objects
-  #class RefundInvoicePayment < Base
-    #belongs_to :invoice_payment
-    #belongs_to :refund
+module Zuora::Objects
+  class RefundInvoicePayment < Base
+    belongs_to :invoice_payment
+    belongs_to :refund
 
-    #validates_presence_of :invoice_payment_id, :refund_amount, :refund_id
+    validates_presence_of :invoice_payment_id, :refund_amount, :refund_id
 
-    #validates_length_of   :created_by_id, :maximum => 32
-    #validates_datetime_of :created_date, :allow_nil => true
-    #validates_length_of   :updated_by_id, :maximum => 32
-    #validates_datetime_of :update_date, :allow_nil => true
+    validates_length_of   :created_by_id, :maximum => 32
+    # validates_datetime_of :created_date, :allow_nil => true
+    validates_length_of   :updated_by_id, :maximum => 32
+    # validates_datetime_of :update_date, :allow_nil => true
 
-    #define_attributes do
-      #read_only :created_by_id, :created_date, :invoice_payment_id, :refund_amount,
-        #:refund_id, :updated_by_id, :updated_date
-    #end
-  #end
-#end
+    define_attributes do
+      read_only :created_by_id, :created_date, :invoice_payment_id, :refund_amount,
+        :refund_id, :updated_by_id, :updated_date
+    end
+  end
+end
```

Comment:
Same again. Do we know what's going on here?

Reply:
Part of the original import from upstream as seen [here](https://github.com/Sage/zuora/commit/35a6399545bd685b5054f1adcd401721e7c35190#diff-8f88ca3e66f063d39f293812a97a9dbf68ceca26a4d592d528f882b6b443e8a0).